index.js

```diff
@@ -167,8 +167,12 @@ if (coreState.argo['local-startup-url']) {
         let localConfig = JSON.parse(fs.readFileSync(coreState.argo['local-startup-url']));
 
         if (typeof localConfig['devtools_port'] === 'number') {
-            console.log('remote-debugging-port:', localConfig['devtools_port']);
-            app.commandLine.appendSwitch('remote-debugging-port', localConfig['devtools_port'].toString());
+            if (!coreState.argo['remote-debugging-port']) {
+                console.log('remote-debugging-port:', localConfig['devtools_port']);
```

Comment:
I think this needs to be `log.writeToLog('info', ' ... ');` for it to actually show up in the logs no?